gotocompany/guardian/v1beta1/guardian.proto

```diff
@@ -569,6 +576,25 @@ message CreateAppealRequest {
   string description = 4;
 }
 
+message PatchAppealRequest {
+  string id = 1;
+  string account_id = 2;
+
+  message Resource {
+    string id = 1;
+    string role = 2;
+    google.protobuf.Struct options = 3;
+    google.protobuf.Struct details = 4;
+  }
+  Resource resource = 3;
```

Comment:
```suggestion
  string resource_id = 1;
  string role = 2;
  google.protobuf.Struct options = 3;
  google.protobuf.Struct details = 4;
```
we can adjust the payload like the above because this is specific for updating one appeal

Reply:
sure, we can update the payload